net-analyzer/httpflow/httpflow-0.0.9.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Analyze HTTP streams from the command line"
+DESCRIPTION="A command line utility helps to capture and dump HTTP stream."
+HOMEPAGE="https://github.com/six-ddc/httpflow"
+COPYRIGHT="2020 dudongcheng"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/six-ddc/httpflow/archive/0.0.9.tar.gz"
+SOURCE_FILENAME="httpflow-$portVersion.tar.gz"
+SOURCE_DIR="httpflow-$portVersion"
+CHECKSUM_SHA256="2347bd416641e165669bf1362107499d0bc4524ed9bfbb273ccd3b3dd411e89c"
+PATCHES="httpflow-${portVersion}.patch"
```

Comment:
Should be a patchset instead of a simple patch. Should come with a header.

Reply:
Also, loose the "{}"

Reply:
👍